packages/@react-spectrum/icon/docs/Icon.mdx

```diff
@@ -25,12 +25,26 @@ category: Misc
 ---
 
 # Icon
-<p>Docs coming soon</p>
+
+<p>{docs.exports.Icon.description}</p>
 
 ## Available Icons
 
 ### Workflow
-<HeaderTable name={workflowIconPackageData.name} version={workflowIconPackageData.version} />
+
+<HeaderInfo
+  packageData={workflowIconPackageData}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/iconography/'}
+  ]} />
+
+<br />
+
+Spectrum workflow icons are graphical metaphors or symbols that users interact with to navigate and manipulate objects.
+They all appear at the same size within each platform, 18 px for desktop and 22 px for mobile. See [ActionButton](/react-spectrum/ActionButton.html#content)
+and [Picker](/react-spectrum/Picker.html#complex-items) for some examples of icon usage within React Spectrum components.
+
+See the table below for a full list of available workflow icons.
 
 ```tsx snippet
 <IconTable iconPackage="workflow" />
```

Comment:
Lemme know if I should add in the prop table w/ examples of how to use `alt`, `size`, `scale` etc. I omitted it for now since we still don't quite support user svgs and didn't want to confuse the people reading the docs with the `children` prop.

Reply:
We could make a how to add custom icons section? won't be important for Adobe people, but definitely important for outside Adobe

I'm happy to go over that with you if we want to do that.